### PR TITLE
Persistant Checkboxes

### PR DESCRIPTION
**Pull request by Carlos-oficial**
Save the currently checked boxes to local storage to be displayed/used after leaving/refreshing the page

### CODE REVIEW

File: components/SelectSchedule.tsx [hunk 1]
```diff
@@ -80,6 +104,18 @@ export const SelectSchedule = ({
   CheckBox creation using Collapse for each subgroup and 
   mapping the values in each array
   */
+
+  const isChecked = (obj: { id, shift }) => {
```
Comment (MarioRodrigues10) [inline components/SelectSchedule.tsx:108]: I think this is a better version for this code
```c
const isChecked = (obj: { id, shift }) => {
  return selectedFilters.some(element => {
    return obj.id === Number(element.id) && obj.shift === element.shift;
  });
}
```

File: components/SelectSchedule.tsx [hunk 2]
```diff
@@ -54,8 +59,27 @@ export const SelectSchedule = ({
   const year_five = filters.filter((f) => f.groupId === 5);
 
   const others = filters.filter((f) => f.groupId === 0);
+  const ordinals = (n) => {
```
Comment (MarioRodrigues10) [inline components/SelectSchedule.tsx:62]: Make this more readble with a switch case

**Review by MarioRodrigues10 — CHANGES_REQUESTED** [review]

**Comment by MarioRodrigues10** [comment]
Please make sure that you also resolve the conflicts :pray:

**Review by diogogmatos — APPROVED** [review]
Good work! :raised_hands:

**Pull request merged**